cheroot/test/conftest.py

```diff
@@ -0,0 +1,38 @@
+import threading
+import time
+
+import pytest
+
+import cheroot.server
+import cheroot.wsgi
+
+
+config = {
+    'bind_addr': ('127.0.0.1', 54583),
```

Comment:
Here I'd like to see support for IPv6 and I'd like not to see hard-coded port numbers... which could conflict with other applications or other instances of the tests running. I suggest using `portend.find_available_local_port` or binding to the ephemeral port 0 and then resolving the bind port for the client.

Reply:
Yeah, I was thinking about it and prefer ephemeral port 0 approach as it's safer to defer port allocation to the OS and `portend` dependency is not even needed in this case.

I was trying to decouple existing `unittest.TestCase` based class into pieces, thus I didn't really take any steps to optimize or add new behavior at this stage. I didn't even replace the test suite module, but added new one because of this experimenting process.